fastapi/testclient.py

```
@@ -0,0 +1,56 @@
+    def text(self) -> str:
+        return self._response.text
+
+    def json(self) -> Any:
+        return self._response.json()
+
+
+class TestClient:
+    def __init__(self, app: FastAPI) -> None:
+        self.app = app
+
+    def _request(self, method: str, path: str, headers: Optional[Dict[str, str]] = None, data: Optional[bytes] = None) -> TestClientResponse:
```

Ruff (0.14.2)

Unused method argument: `data`

(ARG002)